packages/wrangler/src/dev/miniflare.ts

```diff
@@ -837,6 +842,42 @@ export async function buildMiniflareOptions(
 		}
 	}
 
+	const assetServerModulePath = require.resolve(
+		"@cloudflare/workers-shared/dist/asset-server-worker.mjs"
+	);
+	const assetServerConfigPath = require.resolve(
+		"@cloudflare/workers-shared/asset-server-worker/wrangler.toml"
+	);
+	let assetServerConfig: Config | undefined;
+
+	try {
+		assetServerConfig = readConfig(assetServerConfigPath, {});
+	} catch (err) {
+		//  ignore and continue with some predefined default config
```

Comment:
dbl check: is this what we want to do here?

Reply:
Let's just blow up and error? This should never happen, right? We'll ship with that toml, so it should be available. If there's ever an issue there, I'd be happy to have that version of Wrangler or ASW2 package crash out and for us to fix it in a patch.

Reply:
indeed, I agree with you. I feel this should never happen since we own the Worker and should in theory never publish it with an invalid toml file. But even so, having this extra sanity check `try{} catch()` is not a bad  thing IMO. I think it's reasonable to fail here, like you said. Will add that

Reply:
✅